project02.py

```diff
@@ -0,0 +1,27 @@
+GEDCOM_FILE = 'cs555project01.ged'
+VALID_TAGS = ['INDI', 'NAME', 'SEX', 'BIRT', 'DEAT',
+              'FAMC', 'FAMS', 'FAM', 'MARR', 'HUSB', 'WIFE', 
+              'CHIL', 'DIV', 'DATE', 'HEAD', 'TRLR', 'NOTE']
+
+def processFile(file):
+    with open(file, 'r') as f:

```

Comment:
this `open` should be done first, and the strings should be extracted, so that you aren't keeping the file descriptor open the whole time, and to eliminate disk latency